tests/common/include/tests/common/tools_intersection_kernel.inl

```diff
@@ -28,6 +29,9 @@ using namespace detray;
 
 constexpr const float epsilon = 1e-3;
 
+// TODO: How about merging ray and helix tests into one to remove the code
+// repetition?
```

Comment:
Since we are using the helix intersection kernel for the helix now, instead of the normal one, we should indeed do that

Reply:
OK. Let's leave as it is for now